libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -315,13 +315,16 @@ bool ASTJsonConverter::visit(UsingForDirective const& _node)
 	vector<pair<string, Json::Value>> attributes = {
 		make_pair("typeName", _node.typeName() ? toJson(*_node.typeName()) : Json::nullValue)
 	};
+
 	if (_node.usesBraces())
 	{
```

Comment:
We should also change the binaryOperation and unaryOperation ast nodes accordingly!